src/store/profile/actions.ts

```diff
@@ -9,3 +10,76 @@ export const fetchProfile = createAsyncThunk('profile/fetchProfile', async (_, t
 		return thunkAPI.rejectWithValue('Failure');
 	}
 });
+
+export const fetchRequisites = createAsyncThunk('crm/fetchRequisites', async (_, thunkAPI) => {
+	try {
+		const res = await uspacySdk.profileService.getRequisites();
+		return res.data;
+	} catch (e) {
+		return thunkAPI.rejectWithValue('Failure');
+	}
```

Comment:
```suggestion
	} catch (e) {
		return thunkAPI.rejectWithValue(e);
	}
```
Давайте нормально обрабатывать ошибки и так во всём файле